src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -456,16 +456,20 @@ public function cgetNodes(Request $request)
      * @throws MissingParameterException
      * @throws ParameterDataTypeException
      */
-    private function cgetContent(Request $request)
+    protected function cgetContent(Request $request)
```

Comment:
Wasn't it possible to deprecate the entire `NodeController`? Thought of something like this in the constructor:

```
if (get_class($this) === NodeController::class) {
    @trigger_error(...);
}
```

Reply:
Is added